Automattic/wp-calypso
# Add WPCOM_FEATURES_INSTALL_PURCHASED_PLUGINS to the starter plan

#### Proposed Changes

* Add WPCOM_FEATURES_INSTALL_PURCHASED_PLUGINS to the starter plan

#### Testing Instructions

* Apply D82976-code
* Visit `/plugins/<site>` on a simple site with a starter plan.
  * You should be able to buy and install plugins.

This is probably rough and not working properly!

Related to #58998
